apps/site/pages/components/organisms/navbar.mdx

```diff
@@ -0,0 +1,247 @@
+  const navbarPath = path.resolve(__filename)
+  const components = ['Navbar.tsx']
+  const [navbarProps] = getComponentPropsFrom(navbarPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        navbarProps,
+      },
+    },
+  }
+}
+
+export const NavbarPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { navbarProps } = useSSG()
+  return {
+    Navbar: <PropsSection propsList={navbarProps} />,
+  }[component]
+}
+
+<header>
+
+# Navbar
+
+Represents the page header and groups some features like: `Search`, `Regionalization`, `SignIn`, `CartSidebar`.
+
+This component is part of the [Navigation](/components/features/navigation) feature.
+
+</header>
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
+
+```tsx
+import { Navbar } from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/organisms/Navbar/styles.scss'
+```
+
+---
+
+## Usage
+
+Uses `NavbarHeader`, `NavbarRow` and `NavbarButtons` as compound components.
+
+The [NavbarLinks](/components/molecules/navbar-links) and [NavbarSlider](/components/molecules/navbar-slider) components are used as child components.
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection containerStyle={{ display: 'block' }}>
+      <UIProvider>
+        <NavbarUsage />
+      </UIProvider>
+    </OverviewSection>
+  </Tab>
+  <Tab>
+    ```tsx
+      const scrollDirection = useScrollDirection()
+      const { openNavbar, navbar: displayNavbar } = useUI()
+
+      return (
+        <Navbar
+          className="layout__content-full"
+          data-fs-navbar-scroll={scrollDirection}
+        >
+          <NavbarHeader>
+            <NavbarRow className="layout__content">
+              <IconButton
+                data-fs-navbar-button-menu
+                aria-label="Open Menu"
+                icon={<Icon name="List" width={30} height={30} />}
+                onClick={openNavbar}
+              />
+              <IconButton
+                aria-label="Logo"
+                icon={<Icon name="Storefront" width={30} height={30} />}
+              />
+              <SearchInputUsage actions={false} />
+              <NavbarButtons>
+                <LinkButton
+                  href="#"
+                  variant="tertiary"
+                  icon={<Icon name="User" width={18} height={18} weight="bold" />}
+                >
+                  Login
+                </LinkButton>
+                <IconButton
+                  data-fs-cart-toggle
+                  aria-label="cart"
+                  icon={<Icon name="ShoppingCart" width={32} height={32} />}
+                />
+              </NavbarButtons>
+            </NavbarRow>
+          </NavbarHeader>
+
+          <NavbarLinksUsage classes="hidden-mobile" />
```

Comment:
Why not "`className`" here?
```suggestion
          <NavbarLinksUsage className="hidden-mobile" />
```

Reply:
It's just to reproduce the same scenario of the original `NavbarLinks`, some cases we need to use this class and others don't.